UsefulStuff.Core/Features/ImprovedTransformOperations.cs

```diff
@@ -551,60 +565,75 @@ private void UpdateButtonsVisibility()
 
             private void CopyTransform()
             {
-                GuideObject source = _hashSelectObject.First();
-                _savedPosition = source.changeAmount.pos;
-                _savedRotation = source.changeAmount.rot;
-                _savedScale = source.changeAmount.scale;
-                _clipboardEmpty = false;
-                UpdateButtonsVisibility();
+                if (_hashSelectObject.Count == 1)
```

Comment:
Why limit it to only 1 object selected? I suggest at least showing a LogMessage that only 1 can be selected at a time if user tries to copy 2 or more.

Reply:
This is mimicking the fact that the "Copy transform" button is grayed out when more than 1 object is selected, as performed in the `UpdateButtonsVisibility()` function above this one.
Still added the log message to clear that up to when using the shortcuts.